lib/pages/lists.dart

```diff
@@ -44,8 +44,12 @@ class ListsState extends State<Lists> {
                   scrollDirection: Axis.horizontal,
                   itemCount: wishlist.length,
                   itemBuilder: (BuildContext ctxt, int index) {
-                    return WidgetLists(wishlist[index],
-                        "assets/images/basket_my_lists.png", "0");
+                    return GestureDetector(
+                        onTap: () {
+                          openOpenedListPage(context, wishlist[index]);
+                        },
+                        child: WidgetLists(wishlist[index],
+                            "assets/images/basket_my_lists.png", "0 partage"));
```

Comment:
1) why are you giving the picture as parameter ? why don't you use the picture directly inside the WidgetLists widget ?

2) Hard coded string "0 partage" please use getString(...)

Reply:
because after we will change picture for each lists

Reply:
this string will change after